### PR TITLE
research cloud yml files

### PR DESCRIPTION
**Pull request by parisa-zahedi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,23 @@ git clone git@github.com:Semantics-of-Sustainability/tempo-embeddings.git
 cd tempo-embeddings
 python3 -m pip install .
 ```
+## How to use tempo_embeddings on Research Cloud?
+Research Cloud provides a ready-to-use environment for users to run tools without needing to install Python and other required libraries. If your Research Cloud account is set up for the Semantics of Sustainability project, you can use the tempo-embeddings tool by following these steps: can use tempo-embeddings tool with the following steps:
```
Comment (carschno) [inline README.md:41]: Perhaps add a link to accessing/setting up a Research Cloud account, if there is suitable URL?

File: README.md [hunk 2]
```diff
@@ -37,6 +37,23 @@ git clone git@github.com:Semantics-of-Sustainability/tempo-embeddings.git
...
+```
+- Close the terminal, open a new one, and run:
+```console
+cd /scratch/tempo-embeddings/
+conda activate tempo-embeddings
+```
+- Start to play with the notebooks using:
+```console
+jupyter lab notebooks/[1_compute_embeddings_nl.ipynb]
+```
```
Comment (carschno) [inline README.md:56]: For clarification: all these actions are supposed to take place in the Research Cloud environment, right? Perhaps add a comment for clarification.

File: research_cloud/environment.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  - 'deb http://archive.ubuntu.com/ubuntu focal-security main multiverse universe restricted'
+  packages:
+  - ncdu
+
+miniconda:
+  name: tempo-embeddings
+  channels:
+  - conda-forge
+  dependencies:
+  - python=3.9      # change version if needed
```
Comment (carschno) [inline research_cloud/environment.yml:15]: Could we include more recent Python versions?

```suggestion
  - python>=3.9      # change version if needed
```

**Review by carschno — APPROVED** [review]
Only some minor clarification questions and suggestions for potential improvements.

File: README.md (hunk 1 above)
Comment (parisa-zahedi) [reply README.md:41]: Done!

**Comment by parisa-zahedi** [comment]
thanks @carschno for your useful comments. I applied them!

**Review by carschno — APPROVED** [review]

**Pull request merged**